doc/source/methods/IntegratedGradients.ipynb

```diff
@@ -7,6 +7,22 @@
     "# Integrated Gradients"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Note    \n",
+    "\n",
+    "To enable support for Integrated Gradients, you may need to run\n",
+    "\n",
+    "```bash\n",
+    "pip install alibi[tensorflow]\n",
+    "```\n",
+    "\n",
+    "</div>"
```

Comment:
Same comment as above, can we make the box title be "Note" and the text part of the body?